client/login/wp-login/test/test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External Dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal Dependencies
+ */
+import { Login } from '../index.jsx';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+const defaultProps = {
```

Comment:
Defining `defaultProps` here seems kind of weird, I didn't notice other examples of how to do it, though. Should I be loading the `Login` component in some other way that adds the `props` in the normal fashion?